packages/cli/src/requests.ts

```diff
@@ -332,12 +332,7 @@ export declare namespace UserRequest {
 		{ transferId?: string; includeRole: boolean }
 	>;
 
-	export type ChangeRole = AuthenticatedRequest<
-		{ id: string },
-		{},
-		{ newRole?: { scope?: RoleScopes; name?: RoleNames } },
-		{}
-	>;
+	export type ChangeRole = AuthenticatedRequest<{ id: string }, {}, { roleName: RoleNames }, {}>;
```

Comment:
Nit:

```suggestion
	export type ChangeRole = AuthenticatedRequest<{ id: string }, {}, { newRoleName: RoleNames }, {}>;
```

Reply:
Also a union is usually singular: `RoleNames`. But prolly out of scope, up to you.

Reply:
yeah, I'd like to rename all types like these to be singular. but that would be a separate PR. Maybe we can have a linting rule to prevent this from happening again 🤔 ?